### PR TITLE
Ocrvs 2543: Part 1 -  Added ListView Component and Refactored Some Pages with ListView

### PR DESCRIPTION
**Pull request by soumita-dsi**

### CODE REVIEW

File: packages/client/src/views/Home/RecordAudit.tsx [hunk 1]
```diff
@@ -240,6 +240,48 @@ const getLocation = (
   return ''
 }
 
+/*const getSavedApplications = (props: IFullProps): IApplicationData => {
```
Comment (euanmillar) [inline packages/client/src/views/Home/RecordAudit.tsx:243]: @soumita-dsi is this meant to be commented out?

**Review by euanmillar — CHANGES_REQUESTED** [review]
@soumita-dsi can you fix the broken test on this please?

File: packages/client/src/views/Home/RecordAudit.tsx (hunk 1 above)
Comment (soumita-dsi) [reply packages/client/src/views/Home/RecordAudit.tsx:243]: Sorry, I forgot to remove this commented part. I will remove that.

**Comment by soumita-dsi** [comment]
> 
Sure, I will fix them.

**Pull request merged**